src/widgets/tabview/lv_tabview.c

```diff
@@ -43,8 +43,13 @@ const lv_obj_class_t lv_tabview_class = {
     .instance_size = sizeof(lv_tabview_t)
 };
 
-static lv_dir_t tabpos_create;
-static lv_coord_t tabsize_create;
+typedef struct {
+    lv_dir_t tab_pos;
+    lv_coord_t tab_size;
+} lv_tabview_create_info_t;
+
+// only used in lv_obj_class_create_obj, no affect multiple instances
```

Comment:
The only possible issue is when 2 thread are creating a tabview at the same time. However it shouldn't happen as concurrent LVGL function calls should be protected by a mutex. 

Do you see it in the same way?

Reply:
> Yes, I agree. Functions that support reentrancy in a multi-threaded environment should be protected by a mutex.

Reply:
Or move the tab information to the lv_tabview_t structure? Which one do you prefer?